archive/frames/utils.py

```diff
@@ -120,12 +102,8 @@ def build_nginx_zip_text(frames, directory, uncompress=False):
             # portion of the generated URL with an internal NGINX location which proxies all
             # traffic to AWS S3.
             # default location (return files as-is from AWS S3 Bucket)
-            params = {
-                'Key': version.s3_daydir_key,
-                'Bucket': settings.BUCKET,
-            }
-            url = client.generate_presigned_url('get_object', Params=params, ExpiresIn=86400)
-            location = url.replace(f'https://{settings.BUCKET}.s3.amazonaws.com', '/s3-native')
+            url = file_store.get_url(path, version.key, expiration=86400)
+            location = url.replace(f'https://{archive_settings.BUCKET}.s3.amazonaws.com', '/s3-native')
```

Comment:
Is doing zip files on the fly dependent on s3?

Reply:
no, I forgot to update this location thing... I'll have to think about what this should be, since this /s3-native thing is just in our specific nginx deployment....